### PR TITLE
Add scripts for generating protocol buffers.

### PR DESCRIPTION
**Pull request by brendandburns**
@mbohlool (only tested w/ Java)

### CODE REVIEW

File: proto/dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+mkdir -p k8s.io/apimachinery/pkg/api/resource
+mkdir -p k8s.io/apimachinery/pkg/apis/meta/v1
+mkdir -p k8s.io/apimachinery/pkg/util/intstr
+mkdir -p k8s.io/apimachinery/pkg/runtime/schema
+mkdir -p k8s.io/apis/meta/v1
+
+base=https://raw.githubusercontent.com/kubernetes
+
+curl -s ${base}/apimachinery/master/pkg/api/resource/generated.proto \
```
Comment (mbohlool) [inline proto/dependencies.sh]: can we make `master` configurable? the value can come from `settings` file that openapi generator is using.

File: proto/dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo Downloading proto files.
+
+mkdir -p k8s.io/apimachinery/pkg/api/resource
+mkdir -p k8s.io/apimachinery/pkg/apis/meta/v1
+mkdir -p k8s.io/apimachinery/pkg/util/intstr
+mkdir -p k8s.io/apimachinery/pkg/runtime/schema
+mkdir -p k8s.io/apis/meta/v1
```
Comment (mbohlool) [inline proto/dependencies.sh]: are these all generated.proto files we need?

Comment (brendandburns) [reply proto/dependencies.sh]: For the basic V1 API. Doesn't cover any of the other api groups, we can add them later?

File: proto/dependencies.sh (hunk 1 above)
Comment (brendandburns) [reply proto/dependencies.sh]: done.

**Comment by brendandburns** [comment]
Comment addressed, ptal.

Thanks
--brendan

**Pull request merged**